src/sql/helpers.js

```diff
@@ -0,0 +1,95 @@
+const _ = require('lodash');
+
+// Adds commands in a select to transform fields that are in snake_case format to camelCase
+const sqlToCamelCased = (snakeCasedFields) => snakeCasedFields.map((snakecased) => `${snakecased} as ${_.camelCase(snakecased)}`);
```

Comment:
It isn't only transforming snake_case, you can see it also adds the syntax so that sql transforms it:

"samples_order" => "samples_order as samplesOrder"

Reply:
I think we shouldn't be adding this extra logic around the code. afaict we can use something simpler like this https://vincit.github.io/objection.js/recipes/snake-case-to-camel-case-conversion.html

Reply:
It says there that `objection.js` is an ORM, it seems a little overkill to add this package only to use the mapper.

Reply:
oh I see, we could do the camelcase conversion in `postProcessResponse`, good catch!